app/e2e/playwright.config.ts

```diff
@@ -27,7 +27,8 @@ const config: PlaywrightTestConfig = {
     /* Retry on CI only */
     retries: process.env.CI ? 3 : 0,
     /* Amount of workers based on number of browsers being tested */
-    workers: process.env.CI ? 1 : 6,
+    //workers: process.env.CI ? 1 : 6,
+    workers: 2,
```

Comment:
you could just change the 6 to the number of workers you want to run locally instead of commenting out the whole thing. In the future it's fine to change this value locally, but please try to avoid committing those local only changes.